src/app/views/contact/contact.component.js

```diff
@@ -1,5 +1,17 @@
 const template = require('./contact.html');
 
+class contactController {
+  constructor($location) {
+    'ngInject';
+
+    this.$location = $location;
+  }
+
+  go() {
+    this.$location.path('/about').hash('userHelp');
+  }
```

Comment:
I can do this everywhere except in the about component. Because ng-click gets executed before ui-sref or href, we don't get a chance to add location hash before $onChanges is run, and accordion never gets opened. Unless you have another workaround.